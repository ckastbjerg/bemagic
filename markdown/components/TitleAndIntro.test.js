/* eslint-env jest */
const TitleAndIntro = require('./TitleAndIntro');

test('it throws', () => {
    expect(() => TitleAndIntro()).toThrow();
});

test('it works', () => {
    expect(TitleAndIntro({
        componentName: 'block',
        componentClasses: 'prefix-block',
        intro: 'Some intro',
    })).toMatchSnapshot();
});
