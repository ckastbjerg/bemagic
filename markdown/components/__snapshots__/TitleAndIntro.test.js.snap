exports[`test it works 1`] = `
"
        ## Block \`prefix-block\`
        > Some intro
    "
`;
